pkg/stringid/stringid.go

```diff
@@ -58,18 +54,6 @@ func generateID(r io.Reader) string {
 	}
 }
 
-// GenerateRandomID returns a unique id.
-func GenerateRandomID() string {
-	return generateID(cryptorand.Reader)
-}
-
-// GenerateNonCryptoID generates unique id without using cryptographically
-// secure sources of random.
-// It helps you to save entropy.
-func GenerateNonCryptoID() string {
```

Comment:
Should we _deprecate_ this one for the time being, but make it an alias to `GenerateRandomID`? This might be used outside of this project; https://github.com/search?q=GenerateNonCryptoID&type=Code

Reply:
No. We don't version this package, so I don't think it makes sense. It is a one line fix when you re-vendor.